### PR TITLE
Readme file is moved to markdown format

### PR DESCRIPTION
**Pull request by thakurajayL**

### CODE REVIEW

**Comment by omecproject** [comment]
Can one of the admins verify this patch?

File: README.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+- Multiple eNB support is added, but not tested beyond 2 eNB connections.
+
+## Appendix A.
+Before starting openMME applications respective parameters should be  configured in the json files mentioned in section 2.b. For each of the application individual json file can be created, or a user can create a common json file for entire MME configuration(all interfaces), and copy contents of this file in all other jsons.
+	OpenMME applications are designed modularly so that with extended IPC support user can have an individual application running on separate hosts/VMs, to help in this situation separate configuration files are created for each of the MME application(s1ap-app, s6a-app, s11-app, mme-app)
+
+### Config Parameters
+- Following is a description of each section under json file.
+	[U] - Reserved/Unused. Can be skipped in the json.
+	[M] - Mandatory parameter.
```
Comment (krsna1729) [inline README.md]: the newline and indentation is not rendering

File: README.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+#### "mme" - Contains information relevant for mme-app & s1ap-app.
+  - egtp_default_port[U] : Default S11 port for MME's s11 end.
```
Comment (krsna1729) [inline README.md]: Better to make a 3 column table - name, mandator/unused, notes (including defaults if any)

File: README.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+    cd {install_path}/openmme/target/conf
+    ./make_certs.sh <diameter_host> <diameter_realm>
+    For Example: ./make_certs.sh mme localdomain
+
+   ### Start MME modules
+     cd {install_path}/openmme/target/
+     ./run.sh
+
+## Known Issues
```
Comment (krsna1729) [inline README.md:54]: Open github issues instead?

Comment (thakurajayL) [reply README.md:54]: i just ported current README.txt. I am not sure if these issues are still valid. Will figure out if these are still valid issues then will open the issues.

**Comment by micpez** [comment]
I think it is better to move README.txt to README.md, so the change history of the README is preserved.

File: README.md (hunk 2 above)
Comment (thakurajayL) [reply README.md]: Done.

File: README.md (hunk 3 above)
Comment (krsna1729) [reply README.md:54]: Do you want to address this in this PR or follow-on PR?

File: README.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+
+[Config Parameters](#config-parameters)
+
+## Introduction:
+OpenMME is a grounds up implementation of the Mobility management Entity EPC S1 front end to the Cell Tower (eNB). Its design is performance optimized for high speed mobility events over the S1-MME interface, while maintaining state coherent high transaction rate interactions over the S6a interface to the HSS and the S11 interface to the Serving Gateway Control (SGWC). The design allows maximum utilization of the transaction rate allowed by the S1-MME Non-Access Stratum (NAS) messages over SCTP, S6a DIAMETER Attribute Value Pairs (AVPs) over TCP and S11 GTPV2C messages over UDP protocols.
+
+## Build and Installation Procedure
+### Update config 
+Update following config files
+    {install_path}/openmme/src/mme-app/conf/mme.json
```
Comment (krsna1729) [inline README.md:20]: Code block here

File: README.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+#### "mme" - Contains information relevant for mme-app & s1ap-app.
+  Config Parameter Name | Reserved/Unused/Mandatory | Comments |
+  ----------------------|---------------------------|----------|
+  egtp_default_port     | Unused                    | Default S11 port for MME's S11 end |
+  ip_addr               | Mandatory                 | MME's own ip address. Reserved for management. |
+  s1ap_addr             | Unused | MME's s1ap interface IP address |
+  egtp_addr             | Unused | MME's s11 interface IP address |
+  sctp_port             | Unused | MME's sctp(s1ap) port number. |
+  name                  | Unused | MME's name to use in s1 setup with eNB |
+  egtp_default_hostname | Optioal| |
```
Comment (krsna1729) [inline README.md]: typo `Optioal`

File: README.md [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+### Update config 
+Update following config files
+    {install_path}/openmme/src/mme-app/conf/mme.json
+    {install_path}/openmme/src/s1ap/conf/s1ap.json
+    {install_path}/openmme/src/s11/conf/s11.json
+    {install_path}/openmme/src/s6a/conf/*.json
+    update the Diameter Configuration File:
+        {install_path}/openmme/src/s6a/conf/s6a_fd.conf
+       
+- the following values shall be defined:
```
Comment (krsna1729) [inline README.md:27]: Should this be captured in the table?

File: README.md (hunk 3 above)
Comment (thakurajayL) [reply README.md:54]: Need different PR. Will make sure README.md is in good format. I will keep action item on me to verify the issues and open defect.

File: README.md (hunk 4 above)
Comment (thakurajayL) [reply README.md:20]: Done. Looks like GitHub md is bit different that stackedit md. Never mind, I fixed it.

File: README.md (hunk 5 above)
Comment (thakurajayL) [reply README.md]: Fixed.

File: README.md (hunk 3 above)
Comment (thakurajayL) [reply README.md:54]: Opened 2 issues. Other 2 places i need more study and as soon I am clear I will open those issues.

**Comment by thakurajayL** [comment]
> I think it is better to move README.txt to README.md, so the change history of the README is preserved.

SInce file is very small and  whole file is gettig changed. we are going to lose full history.

File: README.md (hunk 6 above)
Comment (thakurajayL) [reply README.md:27]: No..for now let it be like this. Will update it as soon as i understand bit more about these files. net iterations can have better versions of the doc.

File: README.md (hunk 4 above)
Comment (krsna1729) [reply README.md:20]: you can check the preview in the Github web GUI

**Comment by krsna1729** [comment]
Github seems to think there is a conflict. with readme.txt. Can you see why

**Comment by thakurajayL** [comment]
all review comments address. Can someone please approve ?

**Comment by thakurajayL** [comment]
I messed it up.. Raising new request.

**Pull request closed without merging**